keps/sig-node/4540-strict-cpu-reservation/README.md

```diff
@@ -144,11 +139,11 @@ The concern is, when the feature graduates to `Stable`, it will be enabled by de
 
 However, this is exactly the feature intent, best-effort workloads have no KPI requirement, they are meant to consume whatever CPU resources left on the node including starving from time to time. Best-effort workloads are not scheduled to run on the `reservedSystemCPUs` so they shall not be run on the `reservedSystemCPUs` to destablize the whole node.
 
-Nevertheless, risk mitigation has been discussed in details (see archived options below) and we agree to start with the following node metrics of cpu pool sizes in Alpha stage to assess the actual impact in real deployment before revisiting if we need risk mitigation.
+Nevertheless, risk mitigation has been discussed in details (see archived options below) and we agree to start with the following node metrics of cpu pool sizes in Alpha and Beta stages to assess the actual impact in real deployment before revisiting if we need risk mitigation.
```

Comment:
I think we need to call out when we are gonna making decision and sketch out some criterias to understand when "enough" feedback is enough. Otherwise we are painting ourselves in the permabeta corner.

Reply:
ok, I will clarify the wait for feedback is 1 year, we will move the feature to Stable after 1 year.